src/backend/distributed/citus--6.0-3--6.0-4.sql

```diff
@@ -0,0 +1,61 @@
+SET search_path = 'pg_catalog';
+
+CREATE SEQUENCE citus.pg_dist_groupid_seq
+	MINVALUE 1
+	MAXVALUE 4294967296;
+
+CREATE SEQUENCE citus.pg_dist_node_nodeid_seq
+	MINVALUE 1
+	MAXVALUE 4294967296;
+
+ALTER SEQUENCE citus.pg_dist_groupid_seq SET SCHEMA pg_catalog;
+ALTER SEQUENCE citus.pg_dist_node_nodeid_seq SET SCHEMA pg_catalog;
+
+/* add pg_dist_node */
+CREATE TABLE citus.pg_dist_node(
+	nodeid int NOT NULL DEFAULT nextval('pg_dist_groupid_seq') PRIMARY KEY,
```

Comment:
@marcocitus @lithp 
Ops. we have a subtle but not critical bug here. 
We assign `group_id` sequence to `nodeid` column. Similarly, 'nodeid_seq' to `groupid` column.
The output as expected:
```SQL
                            Table "pg_catalog.pg_dist_node"
   Column    |  Type   |                           Modifiers                           
-------------+---------+---------------------------------------------------------------
 nodeid      | integer | not null default nextval('pg_dist_groupid_seq'::regclass)
 groupid     | integer | not null default nextval('pg_dist_node_nodeid_seq'::regclass)
 nodename    | text    | not null
 nodeport    | integer | not null default 5432
 noderack    | text    | not null default 'default'::text
 hasmetadata | boolean | not null default false
Indexes:
    "pg_dist_node_pkey" PRIMARY KEY, btree (nodeid)
    "pg_dist_node_nodename_nodeport_key" UNIQUE CONSTRAINT, btree (nodename, nodeport)
Triggers:
    
```
I'll be opening an issue for this not to lost.